python/ray/_private/log_monitor.py

```diff
@@ -24,8 +24,8 @@
 # entry/init points.
 logger = logging.getLogger(__name__)
 
-# The groups are worker id, job id, and pid.
```

Comment:
do these logging changes need to be done in this PR? unclear why it's required

Reply:
moved to https://github.com/ray-project/ray/pull/22629